src/Order/Single.php

```diff
@@ -285,16 +285,19 @@ function ( $info ) {
 				<label for="postnl_pickup_points"><?php esc_html_e( 'Delivery Date:', 'postnl-for-woocommerce' ); ?></label>
 				<?php
 				foreach ( $filtered_infos as $info_idx => $info_val ) {
+					// Convert to the Dutch date format
+					$dateObj = date_create_from_format('Y-m-d', $info_val);
+					$dutchDate = date_format($dateObj, 'd/m/Y');
```

Comment:
```suggestion
					$date_obj   = date_create_from_format( 'Y-m-d', $info_val );
					$dutch_date = date_format( $date_obj, 'd/m/Y' );
```
We standardize the variable using `snake_case` instead of `camelCase`